### ISSUE TITLE
Add URL via email

### ISSUE THREAD

**Issue by sorenpeter** [omnivore-app/omnivore#1829]
It would be nice if you just send an URL to your omnivore email that it would work just like adding an URL from the app. 
The URL should be in the subject of the mail so it does not get confused.

**Comment by jacksonh** [comment]
Good idea. I guess we can just check the subject and see if its a parsable URL, if so queue a SaveURL operation.

**Comment by sorenpeter** [comment]
Yes that was exactly how I imagined would be the way to implement this.

**Comment by sywhb** [comment]
Hey @sorenpeter, we have implemented this feature and you could save an URL by sending an email to your [Omnivore inbox's email address](https://omnivore.app/settings/emails) and use the URL as the subject.

**Comment by sorenpeter** [comment]
Hi, That sounds awesome and it seem to work as expected too. Than you!

**Comment by sywhb** [comment]
Thanks @sorenpeter for the verification. I will close this for now
